### ISSUE TITLE
[EN][Raid] Atramedes

### ISSUE THREAD

**Issue by paladincnn** [Atlantiss/BugTracker#8586]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: It is now hard to kill Atramedes due to bugs. 

**Current behaviour**: In hardcore mode, when you use an Ancient Dwarven Shield, Nef does not destroy another shield by smelting it. And shields will not reset after wipe/instance reset. Eventually we got only 3 shields usable for this fight. The position of the usable shields changes for every wipe. This makes it very hard to beat the boss.

**Expected behaviour**: Every time players used a shield, Nef should break another one. All shields should reset after wipe.

**Steps to reproduce the problem**:
Get a group for the fight, you will get less usable shields if you used shields during last fight.

**Revision**: Current server version
r5246

**Comment by paladincnn** [comment]
server version was wrong. It should be r5256.

**Comment by Fabiana-B** [comment]
Same on the link, the cutscene with Maloriak and Victor Nefarius about the story of Atramedes is lost too.

**Comment by tomatosz** [comment]
The cutscene only appears when you have done an attempt on Atramedes and than switch difficulty for another attempt.

**Comment by Fabiana-B** [comment]
@tomatosz it is blizzlike?

**Comment by tomatosz** [comment]
No, I mean it works that way on Atlantiss right now. Should have been more clear on that I guess.

Op 6 dec. 2016 om 20:26 heeft Fa <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


@tomatosz<https://github.com/tomatosz> it is blizzlike?

-

**Comment by sunshitwowsucks** [comment]
on normal too shields dont reset unless you change modes and go back in,  very annoying after wiping
